apps/cow-fi/components/ArticlesPageComponents.tsx

```diff
@@ -61,16 +61,23 @@ interface ArticlesPageProps {
   totalArticles: number
   currentPage: number
   allCategories: { name: string; slug: string }[]
+  allArticles?: Article[]
```

Comment:
The only place where this component is in use is ArticlesPage and allArticles prop is always set there. It seems the prop cannot be optional.

Reply:
Addressed